lib/util/package-rules/package-patterns.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { PackagePatternsMatcher } from './package-patterns';
+
+describe('util/package-rules/package-patterns', () => {
+  const packageNameMatcher = new PackagePatternsMatcher();
+
+  describe('match', () => {
+    it('should return false if depName is not defined', () => {
+      const result = packageNameMatcher.matches(
+        {
+          depName: undefined,
+        },
+        {
+          matchPackagePatterns: ['@opentelemetry/http'],
+        }
+      );
+      expect(result).not.toBeNull();
+      expect(result).toBeFalse();
```

Comment:
```suggestion
      expect(result).toBeFalse();
```
i think this should be enough